File: 867/solution.java
class Solution {
    public int[][] transpose(int[][] A) {
        int r = A.length, c = A[0].length;
        int[][] result = new int[c][r];
        for(int i = 0; i < r; i++) {
            for(int j = 0; j < c; j++) {
                result[j][i] = A[i][j];
            }
        }
        
        return result;
    }
}

File: 867/solution.cpp
class Solution {
public:
    vector<vector<int>> transpose(vector<vector<int>>& A) {
        int n = A.size();
        int m = A[0].size();
        vector<vector<int>> result(m, vector<int>(n));
        
        for(int i = 0; i < n; i++) {
            for(int j = 0; j < m; j++) {
                result[j][i] = A[i][j];
            }
        }
        
        return result;
    }
};
